docs/dataset-overviews/co2-isotopes.md

```diff
@@ -19,13 +19,13 @@
 
 The source ID that identifies the dataset to use in the different phases of CMIP7 is given below.
 
-#### CMIP7 AR7 fast track
+#### CMIP7
 
 No data available for this phase yet.
 
-This data is for the CMIP7 AR7 fast track.
-All data sets for use in the fast track are published with a `mip_era` metadata value of 'CMIP7'.
-This metadata value appears in both the file's global metadata as well as its metadata on ESGF.
+This data is for use in CMIP7 production simulations.
```

Comment:
For data that is vapourware, I'd just remove all generic text and rather leave this - "awaiting information from the provider" or something that sounds better

Reply:
Cool, updated, see what you think